### ISSUE TITLE
Slow typing speed (Win 10)

### ISSUE THREAD

**Issue by DOFfactory** [wireapp/wire-desktop#838]
Wire Version: 2.15.2750
Wire for Web Version: 2017.07.18.1250
OS: Windows 10

When typing the "response" of the app is extremely slow. I type quickly, and sometimes I have to wait for several seconds to finish the text in Wire. The speed of the computer in other apps is fine, especially with the i7-7700HQ (Kaby Lake) CPU. 

If I disable the link preview in options, the speed of typing goes back to normal. I have a gut feeling that the culprit may be somewhere in rendering multiple GIFs on UHD screens.

**Comment by FHTheron** [comment]
I see this too and can provide a short sequence to reproduce it:
1. Enter a new message to send in a chat. The message should have many words, so type some 3-letter junk and followed by space and repeat this until the message box has 4 lines (wrapped).
2. Hold down the left arrow on the keyboard - expect the text cursor to move left one character at a time.
3. Note how the cursor updates only every now and then, it's not smooth and really feels like a slow interface.

Note that doing the above with just characters (no spaces) that wrap to 4 lines does not seem to be this slow. I tried with link-preview disabled as well, no difference. I tried to turn off spell-checking, but it looks like the setting is ignored and spelling is always checked. Perhaps then this is the spell-checker making things slow?
